### PR TITLE
New package: VikunjaTeam.VikunjaDesktop version 0.22.0.0

### PR DESCRIPTION
**Pull request by Exorcism0666**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/135165)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping @Exorcism0666, I'll get a notification.
>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Installation failed with exit code -1978335215 2024-01-22 21:00:10.834 [CLI ] Package hash verification failed. SHA256 in manifest [b00597619e8b94401c119b616775e19dcab51f183fa0cc0ca3e4016c69981a18] does not match download [8b48c7d1f2305a11de3f813791077bb2bb7c4f15d05e8bce2db4ab92a4c76185] 2024-01-22 15:02:59.797 [CLI ] Package hash verification failed. SHA256 in manifest [b00597619e8b94401c119b616775e19dcab51f183fa0cc0ca3e4016c69981a18] does not match download [8b48c7d1f2305a11de3f813791077bb2bb7c4f15d05e8bce2db4ab92a4c76185]

(Automated response - build 642.)

File: manifests/v/VikunjaTeam/VikunjaDesktop/0.22.0.0/VikunjaTeam.VikunjaDesktop.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+PackageVersion: 0.22.0.0
+Installers:
+- InstallerUrl: https://dl.vikunja.io/desktop/0.22.0/Vikunja%20Desktop-0.22.0.msi
+  InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: wix
+  InstallerSha256: B00597619E8B94401C119B616775E19DCAB51F183FA0CC0CA3E4016C69981A18
+  ProductCode: '{663DF52A-39F2-4531-A0E5-D22B1B1BC418}'
+- InstallerUrl: https://dl.vikunja.io/desktop/0.22.0/Vikunja%20Desktop-0.22.0.exe
+  Architecture: x86
```
Comment (stephengillie) [inline manifests/v/VikunjaTeam/VikunjaDesktop/0.22.0.0/VikunjaTeam.VikunjaDesktop.installer.yaml:14]:
```suggestion
  Architecture: x64
```

File: manifests/v/VikunjaTeam/VikunjaDesktop/0.22.0.0/VikunjaTeam.VikunjaDesktop.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: VikunjaTeam.VikunjaDesktop
+PackageVersion: 0.22.0.0
+Installers:
+- InstallerUrl: https://dl.vikunja.io/desktop/0.22.0/Vikunja%20Desktop-0.22.0.msi
+  InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: wix
+  InstallerSha256: B00597619E8B94401C119B616775E19DCAB51F183FA0CC0CA3E4016C69981A18
```
Comment (stephengillie) [inline manifests/v/VikunjaTeam/VikunjaDesktop/0.22.0.0/VikunjaTeam.VikunjaDesktop.installer.yaml:11]:
```suggestion

```

File: manifests/v/VikunjaTeam/VikunjaDesktop/0.22.0.0/VikunjaTeam.VikunjaDesktop.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  ProductCode: '{663DF52A-39F2-4531-A0E5-D22B1B1BC418}'
+- InstallerUrl: https://dl.vikunja.io/desktop/0.22.0/Vikunja%20Desktop-0.22.0.exe
+  Architecture: x86
+  InstallerType: nullsoft
+  InstallerSha256: CBECB51C216D49793C245F22BB127814F1805FF6D1C082F761D0F6D113A0C064
```
Comment (stephengillie) [inline manifests/v/VikunjaTeam/VikunjaDesktop/0.22.0.0/VikunjaTeam.VikunjaDesktop.installer.yaml:16]:
```suggestion

```

File: manifests/v/VikunjaTeam/VikunjaDesktop/0.22.0.0/VikunjaTeam.VikunjaDesktop.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  InstallerSwitches:
+    Silent: /?
+    SilentWithProgress: /?
+- InstallerUrl: https://dl.vikunja.io/desktop/0.22.0/Vikunja%20Desktop-0.22.0.zip
+  Architecture: x64
+  InstallerType: zip
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: Vikunja Desktop.exe
+  InstallerSha256: 5158F6B6E2A6F10D5A6C69041316D0A150FC5A05087CFC188A52E95DA441C88D
```
Comment (stephengillie) [inline manifests/v/VikunjaTeam/VikunjaDesktop/0.22.0.0/VikunjaTeam.VikunjaDesktop.installer.yaml:26]:
```suggestion

```

File: manifests/v/VikunjaTeam/VikunjaDesktop/0.22.0.0/VikunjaTeam.VikunjaDesktop.installer.yaml (hunk 2 above)
Comment (stephengillie) [inline manifests/v/VikunjaTeam/VikunjaDesktop/0.22.0.0/VikunjaTeam.VikunjaDesktop.installer.yaml:11]:
```suggestion
  InstallerSha256: 
```

Comment (stephengillie) [inline manifests/v/VikunjaTeam/VikunjaDesktop/0.22.0.0/VikunjaTeam.VikunjaDesktop.installer.yaml:11]:
```suggestion
  InstallerSha256: 8B48C7D1F2305A11DE3F813791077BB2BB7C4F15D05E8BCE2DB4AB92A4C76185
```

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Actual hash:   B00597619E8B94401C119B616775E19DCAB51F183FA0CC0CA3E4016C69981A18
 Expected hash: 8b48c7d1f2305a11de3f813791077bb2bb7c4f15d05e8bce2db4ab92a4c76185

(Automated response - build 642.)

File: manifests/v/VikunjaTeam/VikunjaDesktop/0.22.0.0/VikunjaTeam.VikunjaDesktop.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: VikunjaTeam.VikunjaDesktop
+PackageVersion: 0.22.0.0
+Installers:
+- InstallerUrl: https://dl.vikunja.io/desktop/0.22.0/Vikunja%20Desktop-0.22.0.msi
+  InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: wix
+  InstallerSha256: 8B48C7D1F2305A11DE3F813791077BB2BB7C4F15D05E8BCE2DB4AB92A4C76185
```
Comment (stephengillie) [inline manifests/v/VikunjaTeam/VikunjaDesktop/0.22.0.0/VikunjaTeam.VikunjaDesktop.installer.yaml:11]:
```suggestion
  InstallerSha256: B00597619E8B94401C119B616775E19DCAB51F183FA0CC0CA3E4016C69981A18
```

**Comment by stephengillie** [comment]
> Automatic Validation ended with:
> 
> > Installation failed with exit code -1978335215 2024-01-22 21:00:10.834 [CLI ] Package hash verification failed. SHA256 in manifest [b00597619e8b94401c119b616775e19dcab51f183fa0cc0ca3e4016c69981a18] does not match download [8b48c7d1f2305a11de3f813791077bb2bb7c4f15d05e8bce2db4ab92a4c76185] 2024-01-22 15:02:59.797 [CLI ] Package hash verification failed. SHA256 in manifest [b00597619e8b94401c119b616775e19dcab51f183fa0cc0ca3e4016c69981a18] does not match download [8b48c7d1f2305a11de3f813791077bb2bb7c4f15d05e8bce2db4ab92a4c76185]
> 
> (Automated response - build 642.)

This appears to have reverted.

**Pull request closed without merging**